Jint/Extensions/JavascriptExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text;
+
+namespace Jint.Extensions
+{
+    internal static class JavascriptExtensions
+    {
+        internal static string UpperToLowerCamelCase(this string str)
+        {
+            var sb = new StringBuilder();
```

Comment:
Why not just clone the string and update the first char? That would prevent from building a new StringBuilder on every call too.

Reply:
True, when only simple concatenation of couple of strings happens, StringBuilder can be an overkill 🙂

Reply:
I updated my code. Is this the correct approach?

Reply:
Shortest version below, I guess they don't differ much functionality/performance-wise.

```
internal static string UpperToLowerCamelCase2(this string str)
{
    return char.ToLowerInvariant(str[0]) + str.Substring(1);
}
```

Reply:
I think I allocate 1 string less than you. ToCharArray probably creates a copy too so we both allocate 2 objects.

Too bad we don't use .NET Core 3 D: